transferplugin/transfer

```diff
@@ -189,7 +189,7 @@ def random_string(length):
         s += '0123456789abcdef'[r.randint(0, 15)]
     return s
 
-def default_port(transfer_mode, use_ssl=False):
+def default_port(transfer_mode, use_ssl=True):
```

Comment:
What is the purpose of change `use_ssl` to True by default? 
It seems that I can't get the reason for these changes from the commit message or the title. Not related to remove pre TLS1.2.

Reply:
Consider not to remove the transfer_mode 'http', but set the default to 'https' while use HTTP. when use commond like xe host-call-plugin host-uuid=e9e20414-d522-4c54-9b9a-7eafc6256663   plugin=transfer fn=expose  "args:vdi_uuid=d7ee1875-fb06-4861-89c4-52fe6fb2ff6a"  "args:transfer_mode=http"  , default to use 'https'. and use 'http' by set use_ssl= False .

Reply:
Yes, I mean we can separate this change to another commit with the message above.
Since it's not related to the TLS1.2.